src/lib/zf/zf_stackdump.c

```diff
@@ -26,6 +26,40 @@ static void dump_zockets(SkewPointer<zf_stack> stack, Zocket* zockets,
   }
 }
 
+void dump_attributes(SkewPointer<zf_stack_impl> stimpl)
+{
+  zf_dump("---------------------attributes-------------------------------\n");
+  
+  if(stimpl->sti_tx_ring_max == -1)
+    zf_dump("tx_ring sizeebjtnkjrne=512\n");
+  else
+    zf_dump("tx_ring size=%d\n", stimpl->sti_tx_ring_max);
```

Comment:
Sorry that I didn't think about this before, but I wonder if this should keep the name of the variable, rather than using size? Although perhaps you chose this because the user facing option might use size instead of max? Either way, I think I would prefer that there isn't a space in the middle. (this also applies to `rx_ring size` below)
```suggestion
    zf_dump("tx_ring_max=%d\n", stimpl->sti_tx_ring_max);
```